### ISSUE TITLE
"ValueError: I/O operation on closed file" with pytest since 0.3.5

### ISSUE THREAD

**Issue by The-Compiler** [tartley/colorama#81]
Since upgrading colorama to 0.3.5, I get the following when running [pytest](http://www.pytest.org/) (on Linux):

```
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File ".../python3.4/site-packages/colorama/initialise.py", line 19, in reset_all
    AnsiToWin32(orig_stdout).reset_all()
  File ".../python3.4/site-packages/colorama/ansitowin32.py", line 67, in __init__
    strip = conversion_supported or not is_a_tty(wrapped)
  File ".../python3.4/site-packages/colorama/ansitowin32.py", line 17, in is_a_tty
    return hasattr(stream, 'isatty') and stream.isatty()
ValueError: I/O operation on closed file
```

**Comment by wiggin15** [comment]
I can't reproduce this on Mac or Linux, Python 2.7 and 3.4. However, I think I know what the problem is and how to fix it. Thanks for reporting this!

**Comment by wiggin15** [comment]
Can you please try the latest branch and see if your issue is resolved?

**Comment by The-Compiler** [comment]
This fixes it indeed, thanks for the quick fix! :+1:

Will there be a release with this fix in anytime soon? I'd sure appreciate it!

**Comment by wiggin15** [comment]
I want to wait a while to see if other issues with the latest version pop up

**Comment by tuukkamustonen** [comment]
Just bumped into this. Would be neat to get a fixed release asap (to avoid having to modify the requirements of all our projects).

**Comment by sils1297** [comment]
Need that too

**Comment by wiggin15** [comment]
I released version 0.3.6 with the fix for this issue.

**Comment by toddpi314** [comment]
Confirmed resolve with 0.3.6
👍
